### PR TITLE
GitHub integration doc

### PR DESCRIPTION
**Pull request by alexr00**

### CODE REVIEW

**Comment by alexr00** [comment]
Please feel free to update or add. If there is something that is missing or not receiving enough attention I am happy to add it.

File: docs/editor/github.md [hunk 1]
```diff
@@ -9,8 +9,88 @@ MetaDescription: Working with GitHub in Visual Studio Code
 ---
 # Working with GitHub in VS Code
 
-## Next steps
+Using GitHub with VS Code lets you share your code and collaborate with others. GitHub integration is provided through the GitHub Pull Requests and Issues extension. To get started with the GitHub in VS Code, you'll need to [create an account](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account) and install the [GitHub Pull Requests and Issues extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github). In this topic, we'll demonstrate how you can get some of your favorite parts of GitHub without leaving VS Code.
 
-* [Source Control](/docs/editor/codebasics.md) - Learn about VS Code's source control integration.
+If you're new to source control and want to start there you can [learn about VS Code's source control integration](/docs/editor/codebasics.md).
```
Comment (gregvanl) [inline docs/editor/github.md:14]: /docs/editor/versioncontrol.md?

File: docs/editor/github.md [hunk 2]
```diff
@@ -9,8 +9,98 @@ MetaDescription: Working with GitHub in Visual Studio Code
...
+
+Once you've installed the [GitHub Pull Requests and Issues extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github) you'll need to sign in. Follow the prompts to authenticate with GitHub and return to VS Code.
+
+![Extension Sign In](images/github/extension-signin.png)
+
+## Setting up a repository
+
+### Cloning a repository
+
+You can search for and clone a repository from GitHub using the **Git: Clone** command or by using the **Clone Repository** button in the Source Control (available when you have no folder open).
```
Comment (joaomoreno) [inline docs/editor/github.md:26]:
```suggestion
You can search for and clone a repository from GitHub using the **Git: Clone** command or by using the **Clone Repository** button in the Source Control (available when you have no folder open or in the command palette via <kbd>F1</kbd>).
```

**Comment by alexr00** [comment]
Thank you for the changes and feedback!

**Pull request merged**